test/testSelenium.js

```diff
@@ -1,89 +1,106 @@
-var assert = require('assert');
-var webPush = require('../index');
-var createServer = require('./helpers/create-server');
-var isPortOpen = require('./helpers/port-open');
-
-webPush.setGCMAPIKey('AIzaSyAwmdX6KKd4hPfIcGU2SOfj9vuRDW6u-wo');
-
-process.env.PATH = process.env.PATH + ':test_tools/';
-
-suite('selenium', function() {
-  this.timeout(180000);
+'use strict';
 
+(function() {
   var invalidNodeVersions = /0.(10|12).(\d+)/;
   if (process.versions.node.match(invalidNodeVersions)) {
-    console.log('Skipping selenium tests as they can\'t run on ' + process.versions.node);
+    console.log('Skipping downloading browsers as selenium tests can\'t run on ' + process.versions.node);
     return;
   }
 
-  var webdriver = require('selenium-webdriver');
-  var firefoxBrowsers = require('./browser-managers/firefox-browsers.js');
-  var chromeBrowsers = require('./browser-managers/chrome-browsers.js');
+  const seleniumAssistant = require('selenium-assistant');
+  const webdriver = require('selenium-webdriver');
+  const seleniumFirefox = require('selenium-webdriver/firefox');
+  const assert = require('assert');
+  const mkdirp = require('mkdirp');
+  const fs = require('fs');
+  const del = require('del');
+  const chalk = require('chalk');
+  const webPush = require('../index');
+  const createServer = require('./helpers/create-server');
+  const isPortOpen = require('./helpers/port-open');
 
-  var VAPID_PARAM = {
+  webPush.setGCMAPIKey('AIzaSyAwmdX6KKd4hPfIcGU2SOfj9vuRDW6u-wo');
+
+  const PUSH_TEST_TIMEOUT = 120 * 1000;
+  const VAPID_PARAM = {
     audience: 'https://www.mozilla.org/',
     subject: 'mailto:web-push@mozilla.org',
     privateKey: new Buffer('H6tqEMswzHOFlPHFi2JPfDQRiKN32ZJIwvSPWZl1VTA=', 'base64'),
     publicKey: new Buffer('BIx6khu9Z/5lBwNEXYNEOQiL70IKYDpDxsTyoiCb82puQ/V4c/NFdyrBFpWdsz3mikmV6sWARNuhRbbbLTMOmB0=', 'base64'),
   };
-  var globalServer, globalDriver;
+  const testDirectory = './test/output/';
 
-  suiteSetup(function() {
-    this.timeout(0);
+  let globalServer, globalDriver, testServerURL;
 
-    var promises = [];
-    promises.push(firefoxBrowsers.downloadDependencies());
-    promises.push(chromeBrowsers.downloadDependencies());
+  function runTest(browser, options) {
+    options = options ? options : {};
 
-    return Promise.all(promises)
-    .then(function() {
+    if (browser.getSeleniumBrowserId() === 'firefox' &&
+      browser.getVersionNumber() < 49) {
```

Comment:
Yeah, I've found a way to do it. Instead of `window.subscribeSuccess` in testSelenium.js you can use `window.wrappedJSObject.subscribeSuccess` (when `wrappedJSObject is there).